downstream/modules/platform/proc-running-setup-script.adoc

```diff
@@ -39,4 +39,4 @@ endif::mesh-VM[]
 
 .Additional resources
```

Comment:
I can't see the entire code, but do you already have [role="_additional-resources"] above line 40?